qemu/tests/blockdev_inc_backup_rm_persistent_bitmap.py

```diff
@@ -0,0 +1,78 @@
+from provider.block_dirty_bitmap import get_bitmap_by_name
+from provider.block_dirty_bitmap import block_dirty_bitmap_disable
+from provider.block_dirty_bitmap import block_dirty_bitmap_remove
+from provider.blockdev_live_backup_base import BlockdevLiveBackupBaseTest
+
+
+class BlockdevIncbkRmPersistentBitmapTest(BlockdevLiveBackupBaseTest):
+    """Persistent bitmaps remove testing"""
+
+    def disable_bitmaps(self):
+        list(map(
+            lambda n, b: block_dirty_bitmap_disable(self.main_vm, n, b),
+            self._source_nodes, self._bitmaps))
+        self._disabled_bitmaps_info = self.get_bitmaps()
+
+    def get_bitmaps(self):
+        return list(map(
+            lambda n, b: get_bitmap_by_name(self.main_vm, n, b),
+            self._source_nodes, self._bitmaps))
+
+    def remove_bitmaps(self):
+        list(map(
+            lambda n, b: block_dirty_bitmap_remove(self.main_vm, n, b),
+            self._source_nodes, self._bitmaps))
+
+    def powerdown_and_start_vm(self):
+        self.main_vm.monitor.system_powerdown()
+        if not self.main_vm.wait_for_shutdown(
+                self.params.get_numeric("shutdown_timeout", 360)):
+            self.test.fail("Failed to poweroff vm")
+        self.main_vm.create()
+        self.main_vm.verify_alive()
+
+    def check_image_bitmaps_gone(self):
+        """bitmaps should be removed"""
+        def _check(tag):
+            out = self.source_disk_define_by_params(self.params, tag).info()
+            if out:
+                if self.params['check_bitmaps'] in out:
+                    self.test.fail(
+                        'Persistent bitmaps should be gone in image')
+            else:
+                self.test.error('Error when querying image info with qemu-img')
+
+        list(map(_check, self._source_images))
+
+    def do_test(self):
+        self.do_full_backup()
+        self.generate_inc_files()
+        self.disable_bitmaps()
+        self.powerdown_and_start_vm()
+        self.remove_bitmaps()
+        self.main_vm.destroy()
+        self.check_image_bitmaps_gone()
+
+
+def run(test, params, env):
+    """
+    Persistent bitmaps removal testing
+
+    test steps:
+        1. boot VM with a 2G data disk
+        2. format data disk and mount it, create a file
+        3. add target disks for backup to VM via qmp commands
+        4. do full backup and add persistent bitmap
+        5. create another file
+        6. disable bitmaps, record the count of bitmaps
+        7. shutdown&restart vm, check count of bitmaps should keep the same
```

Comment:
There is no bitmap check after restart the VM, shall we add it?

Reply:
It used to check the bitmaps after vm restart, then I removed the check for #2552 covered this part. Just updated the test steps, please make a double check, thanks